### DISCUSSION TITLE
Should we start nesting docs links in /docs?

### DISCUSSION THREAD

**Discussion by Toflar** [contao/to.contao.org#81] [Polls]
See https://github.com/contao/to.contao.org/pull/80

**Comment by leofeyer** [comment]
If we decide to nest docs links, we also have to adjust the existing ones:

* `/manager-password`: deep link to https://docs.contao.org/manual/en/installation/contao-manager/#did-you-forget-your-contao-manager-login-information
* `/shortcuts`: deep link to https://docs.contao.org/manual/en/administration-area/back-end-keyboard-shortcuts/
* `/webroot`: deep link to https://docs.contao.org/manual/en/installation/install-contao/#hosting-configuration
* `/system-requirements`: deep link to https://docs.contao.org/manual/en/installation/system-requirements/
* `/quickstart`: deep link to https://docs.contao.org/manual/en/installation/quickstart/

I would rather not have half of the links without and the other half with the `/docs` prefix.

**Reply by Toflar** [reply]
We have to leave them for now (at least for an interim phase). I would update https://github.com/contao/to.contao.org/pull/80 with new redirects, if that's okay? So e.g. `/manager-password` would go to `/docs/manager-password` and this one then to the existing deeplinks?

**Reply by leofeyer** [reply]
Yes, that sounds reasonable. 👍

**Comment by Toflar** [comment]
The result is pretty clear, I have updated https://github.com/contao/to.contao.org/pull/80 accordingly.
